### DISCUSSION TITLE
Failed to setup Copilot in Neovim

### DISCUSSION THREAD

**Discussion by rankun203** [community/community#47542] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I installed the Copilot extension following this instruction https://github.com/github/copilot.vim, then ran

```bash
:Copilot setup
```

It automatically prints

```bash
Authenticated as GitHub user rankun203
```

Then I go to a piece of code and Tab, nothing happened, I explicitly run `:Copilot`, it says

```bash
Not authenticated: NotSignedIn
```

----

Solution

I think it's my local env messing up with extension login script, I disabled this piece of checkStatus and force it to manually login, it worked.

<img width="746" alt="image" src="https://user-images.githubusercontent.com/2988555/219009710-5d9a02da-bd56-4c6d-a809-30528fb8d9ae.png">

**Comment by unknowVariable** [comment]
Thanks for sharing your experience with the Copilot extension! It's great to hear that you were able to find a solution to the issue you were experiencing.

It's possible that there was an issue with the login script or that your local environment was interfering with the authentication process. Disabling the checkStatus and manually logging in is a good workaround, but it's worth noting that modifying the extension's code can have unintended consequences and may affect its behavior in other ways.

If you continue to experience issues with the Copilot extension or other tools, I would recommend reaching out to the support team for that tool or platform for assistance. They may be able to provide guidance or help troubleshoot any issues you're encountering. Additionally, you can also check for any available documentation or online forums to see if others have experienced similar issues and have found solutions.

**Comment by iamcjwilliams** [comment]
@rankun203 Visit [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue

**Comment by rankun203** [comment]
Hi, I tried when it was failing

```
:Copilot status
```

will show it's logged in as rankun203

But when I try to run `:Copilot` to do some magic, it shows not logged in.

**Reply by rankun203** [reply]
The only way I was able to get around is to do as I suggested above, then login again and succeeded, then `Copilot status` works and `:Copilot` works too, and auto completion works too.

**Reply by rankun203** [reply]
The comment which were marked as off topic smells like "ChatGPT".

**Reply by TimAnthonyAlexander** [reply]
Well what worked now? What was suggested above?

**Comment by naknomum** [comment]
fwiw, i had **the exact same problem**.  it showed me as `Authenticated as GitHub user naknomum` but then nothing worked.  i followed your suggestion of commenting out the bit of code and _then i got redirected to github to type the code and **everything works** now_.

i wonder if this has something to do with the fact that:
1. i had previously tried copilot for the free trial (same user)
2. now have a paid account
3. was also (previouly) running an old version of `copilot.vim`

but thanks for the workaround. would be nice if this bug was fixed. sucks to **pay for copilot** and then not be able to use it.

**Comment by buscon** [comment]
hi,

I am having exactly the same problem but and the suggested solution work for me too.
